ruby/lib/google/protobuf/repeated_field.rb

```diff
@@ -32,9 +32,26 @@
 module Google
   module Protobuf
     class RepeatedField
+      extend Forwardable
+
+      def_delegators :to_ary,
+        :bsearch, :compact, :fetch, :inspect,
```

Comment:
definitely.  The downside is I'll need to hand-craft the `#!` methods.  Not terrible but there it is.

let me go ahead and do that (esp. if you are in agreement with my gist where I wrote up some TODOS), and tests will definitely be added.  thanks!